etc/get_dep_version.py

```diff
@@ -0,0 +1,21 @@
+import json
```

Comment:
slightly more general version of the one I wrote for the driver so it can be used for an arbitrary dependency.